### PR TITLE
Add new page for governance info

### PR DESCRIPTION
**Pull request by crandmck**
This is for #632.  I created a new page "Contributing to Express" and copied into it info from 3 files in `expressjs/express`: 
- https://github.com/expressjs/express/blob/master/Collaborator-Guide.md
- https://github.com/expressjs/express/blob/master/Contributing.md
- https://github.com/expressjs/express/blob/master/Security.md
  
  I also consolidated some info in this repo that was duplicated between README.md and CONTRIBUTING.md.

### CODE REVIEW

**Comment by crandmck** [comment]
@expressjs/express-tc PTAL.
We might want to add some links to other docs in the Node Foundation.  Open to suggestions.

File: en/resources/contributing.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+not to add this new member. Members who do not participate consistently at the level of 
+a majority of the other members are expected to resign.
+
+## Express collaborator guide
+
+> NOTE: This is copied from the [Collaborator guide in the repo](https://github.com/expressjs/express/blob/master/Collaborator-Guide.md).
+
+### Website Issues
+
+Open issues for the expressjs.com website in https://github.com/strongloop/expressjs.com.
```
Comment (blakeembrey) [inline en/resources/contributing.md]: I think we should change `strongloop` to `expressjs` now.

**Comment by crandmck** [comment]
Good catch @blakeembrey I made that change.  I'll open a separate PR to change https://github.com/expressjs/express/blob/master/Collaborator-Guide.md which has the same mistake.

**Comment by dougwilson** [comment]
Looks good overall.

**Comment by crandmck** [comment]
I'll go ahead and merge, and we can iterate further if desired.

**Pull request merged**